/**:
  ros__parameters:
    map_path: '/home/oliver/ma_prediction_integration/src/maps/sample-map-planning-modi/lanelet2_map.osm'
    input_topic_objects: '/perception/object_recognition/tracking/objects'
    output_topic_objects: '/perception/object_recognition/objects'
    pareller_output_topic: '/parellel/objects'
    time_horizon: 10.0
    sampling_time_interval: 0.5
    min_crosswalk_user_velocity: 1.0